### PR TITLE
Entry view

### PR DESCRIPTION
**Pull request by SergeBrakow**
Creating an entry card. It is shown in home. 
Create navigation to EditOrder / ViewOrder (these sites are not implemented yet, so there is just a message).

Fix Error page navigation path.

### CODE REVIEW

File: src/components/EntryCard.js [hunk 1]
```diff
@@ -14,9 +37,52 @@ export const StyledCard = styled.div`
     border: solid 1px;
```
Comment (s-kond) [inline src/components/EntryCard.js:37]: I suggest to make every card an 'article', that would be more semantic than a div

**Review by s-kond — COMMENTED** [review]
View and Edit-Buttons work fine 👍 But when I create a new entry, it doesn't appear in home - is that intended at this stage or is something missing?

And some some small comments here:

File: src/components/EntryCard.js (hunk 1 above)
Comment (SergeBrakow) [reply src/components/EntryCard.js:38]: you´re right. fixed now

File: src/components/EntryCard.js [hunk 2]
```diff
@@ -1,11 +1,34 @@
...
+                    <NoteDiv>
+                        <div>
+                            <PBoltInShow>Notizen</PBoltInShow>
+                            <p>{entry.note}</p>
+                            <PBoltInShow>verlinkte Kunden</PBoltInShow>
+                            <PBoltInShow>verlinkte User</PBoltInShow>
+                            <p>{entry.owner}</p>
+                        </div>
+                    </NoteDiv>
+                    :""}
```
[the remark this answers is not in the document]
Comment (SergeBrakow) [reply src/components/EntryCard.js]: nice. thx

**Comment by SergeBrakow** [comment]
> View and Edit-Buttons work fine 👍 But when I create a new entry, it doesn't appear in home - is that intended at this stage or is something missing?
> 
> And some some small comments here:

i fixed the problem. It was not completely ready, yet.

**Pull request merged**